.gitignore

```diff
@@ -33,3 +33,25 @@ yarn-error.log*
 #yarn
 .yarn/*
 !.yarn/releases
+
+#amplify-do-not-edit-begin
+amplify/\#current-cloud-backend
+amplify/.config/local-*
+amplify/logs
+amplify/mock-data
+amplify/mock-api-resources
+amplify/backend/amplify-meta.json
+amplify/backend/.temp
+build/
+dist/
+node_modules/
+aws-exports.js
```

Comment:
@SovereignAndrey , since `aws-exports` is not included in our source control , CI encounters this error
`Module not found: Error: Can't resolve './aws-exports' in '/home/runner/work/angelprotocol-web-app/angelprotocol-web-app/src'`

I tried to add some commands to `amplify.yaml` per docs
but doesn't work :( 

maybe need this step ? 
https://docs.aws.amazon.com/amplify/latest/userguide/amplify-config-autogeneration.html

Reply:
@ap-justin Hmm... I thought we  had the continuous deployments and builds setup. Will check if it's not an issue with IAM Role for Amplify. :+1:

Reply:
@ap-justin OK! I added `AmplifyBackendDeployFullAccess` to the existing Amplify Service Role that is assigned. See if that works for ya. :crossed_fingers:

Reply:
Ah, I think I understand now. It's the GH CI that's the issue, now AWS. We will prob need to setup a GH CI user. On it!